server.cfg

```
@@ -507,6 +507,8 @@ sets Discord "https://discord.gg/usarrp"
 
 set sv_enforceGameBuild 2699
 
+sv_pureLevel 1
```

```suggestion
sets sv_pureLevel 1
```

as defined in the forum post

---

https://forum.cfx.re/t/cfx-re-community-update-june-2022-edition/4866955